OpenEdXMobile/res/values/strings.xml

```diff
@@ -604,5 +606,9 @@
     <string name="share_accomplishment_button">Share your accomplishment</string>
     <!-- Text that pre-populates your post/email/SMS when sharing certificate -->
     <string name="share_accomplishment_message">I just earned a badge on {platform_name}! Check it out: {badge_url}</string>
-
+    <string name="rating_dialog_message">How would you rate the edX app?</string>
+    <string name="rate_app_dialog_title">Rate the app</string>
+    <string name="rate_app_dialog_message">Tell others what you think of the edx app by writing a quick review in the app store</string>
```

Comment:
> Though, is it necessary to name the app store/play store? How about just ending the sentence after "...by writing a quick review"?

I'd second that, cuz in Android we can have a store other than PlayStore like in China they have Baidu and similar stores for Android apps.